documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -56,6 +56,10 @@ You can have up to 100 `include` and `exclude` rules combined. Generally you can
 
 Preferences for the emulated `platform.env` local bindings. See the [getPlatformProxy](https://developers.cloudflare.com/workers/wrangler/api/#syntax) Wrangler API documentation for a full list of options.
 
+#### persist
+
+Indicates if and where to persist the bindings data. If not present or true, defaults to the same location used by Wrangler, so data can be shared between it and the caller. If false, no data is persisted to or read from the filesystem.
```

Comment:
"not present or true" is confusing because I don't know if "not" applies only to "present" or also to "true"
```suggestion
Indicates if and where to persist the bindings data. If `true` or `undefined`, defaults to the same location used by Wrangler, so data can be shared between it and the caller. If false, no data is persisted to or read from the filesystem.
```